cartridges/int_affirm/cartridge/scripts/data/affirmData.js

```diff
@@ -38,6 +38,16 @@
                 currentSite.getCustomPreferenceValue('AffirmOnline') :
                 false;
         };
+        /**
+         * Return Affirm payment country in ISO 3166-1 Alpha-3 code
+         *
+         * @returns {string} Country code
+         */
+        this.getCountryCode = function () {
+            return !empty(currentSite.getCustomPreferenceValue('AffirmPaymentCountryCode')) ?
+                currentSite.getCustomPreferenceValue('AffirmPaymentCountryCode').getValue() :
+                'USA';
```

Comment:
can this be a global config or constant like default country, available country, etc, would recommend everywhere in the code to refer to that than hard-coding USA & CAN?

Reply:
Agreed. The list of countries is included in the metadata import file (and the line 49 is intended in case the merchant does not update the import file of the metadata.): https://github.com/Affirm/Salesforce_Commerce_Cloud_Affirm/blob/ca-support/metadata/affirm/meta/system-objecttype-extensions.xml#L419-L435